niap-cc/Permissions/TransactIds/app/src/main/java/com/android/certifications/niap/permissions/transactids/ProxyChecker.java

```diff
@@ -13,8 +13,8 @@
 import java.util.stream.Collectors;
 
 public class ProxyChecker {
-    private static String TAG="tag";
-    private static Map<String,String> alt_descriptor= new HashMap<String,String>();
+    private static final String TAG="tag";
```

Comment:
Should this tag be set to the class name so it's easier to identify trace entries?